tools/lkl/lib/hijack/hijack.c

```diff
@@ -283,3 +283,20 @@ int epoll_ctl(int epollfd, int op, int fd, struct epoll_event *event)
 
 	return lkl_call(__lkl__NR_epoll_ctl, 4, epollfd, op, fd, event);
 }
+
+int eventfd(unsigned int count, int flags)
+{
+	return lkl_sys_eventfd2(count, flags);
```

Comment:
Oh, but regardless --- `eventfd` will never need to touch the host, since it creates a new fd; there's no way for the user to ask it to hit the host (except perhaps by adding our own flag). This is a useful discussion but it doesn't seem to apply here.